locations/spiders/pizza_hut_be.py

```diff
@@ -0,0 +1,47 @@
+import scrapy
+
+from locations.hours import DAYS, OpeningHours
+from locations.items import Feature
+
+
+class PizzaHutBESpider(scrapy.Spider):
+    name = "pizza_hut_be"
+    item_attributes = {"brand": "Pizza Hut", "brand_wikidata": "Q191615"}
+    start_urls = ["https://restaurants.pizzahut.be/api/stores"]
+    custom_settings = {
+        "ROBOTSTXT_OBEY": False,
+    }
+
+    def parse(self, response):
+        for data in response.json():
+            yield scrapy.Request(f"https://api.pizzahut.be/stores/{data.get('id')}", callback=self.parse_store)
+
+    def parse_store(self, response):
+        oh = OpeningHours()
+        store = response.json().get("store")
+        for ohs in store.get("openingHours"):
+            open, close = ohs.get("shift1").split("/")
+            oh.add_range(day=DAYS[ohs.get("dayNum") - 1], open_time=open, close_time=close)
+
+        address_details = store.get("address")
+        properties = {
+            "ref": store.get("code"),
+            "addr_full": " ".join(
```

Comment:
IMO a comma separator is better. For people and for computers.

```suggestion
            "addr_full": ", ".join(
```